cube/assets/shader.f.glsl

```diff
@@ -0,0 +1,11 @@
+#version 100
```

Comment:
lol is it redundant to have shader in front? I figure eventually there will be multiple shaders (is that a thing?) then they can be named more appropriately.